doc/Process_Console.md

```diff
@@ -600,3 +603,47 @@ tock$
       0x00040800 ┴─────────────────────────────────────────── H
 
 ```
+
+### `commands history`
+ - You can use the up and down arrows to scroll through the command history and to view the previous commands you have run.
+ - If you inserted more commands than the command history can hold, oldest commands will be overwritten.
+ - You can view the commands in bidirectional order, `up arrow` for oldest commands and `down arrow` for newest.
+ - If the user custom size for the history is set to `0`, the history will be disabled and the rust compiler will be able to optimize the binary file by removing dead code.
+
+  Here is how to add a custom size for the command `history` used by the ProcessConsole structure to keep track of the typed commands, in the `main.rs` of boards:
+ ```rust
+ const CUSTOM_HISTORY_LEN : usize = 30;
+
+ /// ...
+ 
+ pub struct Platform {
+    // other fields
+    
+    pconsole: &'static capsules::process_console::ProcessConsole<
+        'static,
+        { CUSTOM_HISTORY_LEN },
```

Comment:
I would rename it to "COMMAND_HISTORY_LEN" to be more suggestive.

Reply:
Ok, I will modify this in the next commits

Reply:
Done